scripts/run_commands.sh

```diff
@@ -0,0 +1,92 @@
+#
+# This scripts runs on the ec2 instance. It is uploaded to the EC2 instance by
+# test_commands.sh
+# The necessary env variables will be set by test_commands.sh using the
+# command line arguments provided to it. 
+#
+
+declare -i FAILURES
+FAILURES=0
+
+RED='\033[1;31m'
+GREEN='\033[1;32m'
+LRED='\033[0;31m'
+LGREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+expect_success() {
+	if "${@}" &>> ~/test_output.txt; then
+		echo -e "${GREEN}SUCCEEDED${NC}: ${LGREEN}${@}${NC}"
+	else
+		echo "-------"
+		echo -e "${RED}FAILED${NC}: ${LRED}${@}${NC}"
+		echo "-------"
+		FAILURES+=1
+	fi
+}
+
+# used for commands that whose arguments might contain sensitive information
+expect_success_no_log() {
+	if "${@}" &>> ~/test_output.txt; then
+		echo -e "${GREEN}SUCCEEDED${NC}: ${LGREEN}${1} ${2}${NC}"
+	else
+		echo "-------"
+		echo -e "${RED}FAILED${NC}: ${LRED}${1} ${2}${NC}"
+		echo "-------"
+		FAILURES+=1
+	fi
+}
+
+# install git and go
+echo "TYPE y|yes then enter to proceed."
+sudo yum install git go >> ~/test_log.txt
+# have to respond yes to prompt
+# get CLI
+export GOPATH="$HOME/go"
+go get github.com/aws/amazon-ecs-cli >> ~/test_log.txt
+cd go/src/github.com/aws/amazon-ecs-cli
+url="https://github.com/"
+url+=$gitname
+url+="/amazon-ecs-cli.git"
+git remote add fork $url &>> ~/test_log.txt
+git fetch fork &>> ~/test_log.txt
```

Comment:
We need a better home for these temp files, should we move this to ~/tmp?

Reply:
That's a good idea. I think I'd prefer to name it `~/test-results/` or something like that.

Reply:
to be more specific, what do you think of using `ecs-cli-test-results`